### PR TITLE
roboto: change license from apache to ofl

### PR DESCRIPTION
**Pull request by m4rc1e**
This PR takes the static fonts we currently serve, v2.317 and moves them to the ofl dir in order to start the license change process. I took the Roboto static fonts from the Google Fonts download zip.

I'll block #6943 not since this PR must come first.

cc @chrissimpkins

### CODE REVIEW

**Comment by emmamarichal** [comment]
@chrissimpkins can I/you merge this PR in order to unblock #6943?

**Comment by m4rc1e** [comment]
@emmamarichal leave this pr for now. I need to talk to the backend team. We may not need to do anything and this pr will be closed.

File: ofl/roboto/OFL.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
-Copyright 2011 The Roboto Project Authors (https://github.com/google/roboto)
+Copyright 2011 Google Inc. All Rights Reserved.
 
 This Font Software is licensed under the SIL Open Font License, Version 1.1.
 This license is copied below, and is also available with a FAQ at:
-http://scripts.sil.org/OFL
+https://scripts.sil.org/OFL
```
Comment (emmamarichal) [inline ofl/roboto/OFL.txt:5]: Following https://github.com/google/fonts/issues/7062 the OFL url should be updated to https://openfontlicense.org/

File: ofl/roboto/METADATA.pb [hunk 2]
```diff
@@ -1,47 +1,121 @@
 name: "Roboto"
-designer: "Christian Robertson, Paratype, Font Bureau"
+designer: "Christian Robertson"
 license: "APACHE2"
```
Comment (emmamarichal) [inline ofl/roboto/METADATA.pb:3]: should be "OFL" right?

**Review by emmamarichal — CHANGES_REQUESTED** [review]

**Comment by emmamarichal** [comment]
@m4rc1e can we close this one, since you open a new one https://github.com/google/fonts/pull/7231 ?

**Comment by m4rc1e** [comment]
Closing in favour of #7231

**Pull request closed without merging**